### PR TITLE
config: fix cln listconfig output

### PR DESCRIPTION
**Pull request by grubles**
CLN PeerSwap `listconfig` currently outputs a `liquid.disabled` field, which doesn't match how `liquidswaps` is set in the `peerswap.conf` file. The output will print as the opposite of how `liquidswaps` is set. This should fix it to properly match how the config option is set.

### CODE REVIEW

File: clightning/clightning.go [hunk 1]
```diff
@@ -356,7 +356,7 @@ func (cl *ClightningClient) SetPeerswapConfig(config *Config) {
 		LiquidRpcHost:          config.Liquid.RpcHost,
 		LiquidRpcPort:          config.Liquid.RpcPort,
 		LiquidRpcWallet:        config.Liquid.RpcWallet,
-		LiquidDisabled:         *config.Liquid.LiquidSwaps,
+		LiquidEnabled:          *config.Liquid.LiquidSwaps,
```
Comment (YusukeShimizu) [inline clightning/clightning.go:359]: [imo]How about the same name?
```suggestion
		LiquidSwaps:          *config.Liquid.LiquidSwaps,
```

File: clightning/options.go [hunk 2]
```diff
@@ -10,7 +10,7 @@ const (
 	liquidRpcUserOption             = "peerswap-elementsd-rpcuser"
 	liquidRpcPasswordOption         = "peerswap-elementsd-rpcpassword"
 	liquidRpcPasswordFilepathOption = "peerswap-elementsd-rpcpasswordfile"
-	liquidDisabledOption            = "peerswap-elementsd-disabled"
+	liquidEnabledOption             = "peerswap-elementsd-enabled"
```
Comment (YusukeShimizu) [inline clightning/options.go:13]: Same background.
How about the same name?
```suggestion
	liquidSwapsOption             = "peerswap-elementsd-swaps"
```

File: clightning/options.go [hunk 3]
```diff
@@ -53,7 +53,7 @@ type PeerswapClightningConfig struct {
 	LiquidRpcHost         string `json:"liquid.rpchost"`
 	LiquidRpcPort         uint   `json:"liquid.rpcport"`
 	LiquidRpcWallet       string `json:"liquid.rpcwallet"`
-	LiquidDisabled        bool   `json:"liquid.disabled"`
+	LiquidEnabled         bool   `json:"liquid.enabled"`
```
Comment (YusukeShimizu) [inline clightning/options.go:56]: ditto
```suggestion
	LiquidSwaps         bool   `json:"liquid.swaps"`
```

**Review by YusukeShimizu — APPROVED** [review]
It does not appear that anything will be broken by this change.

The `legacyOptions` in `clightning/options.go` are `deprecated` and may be deleted separately

Also, I will respond to the missing config console print setting in the lwk response.

**Comment by grubles** [comment]
Should we go ahead and remove the `legacyOptions` in this PR? What about `RegisterOptions()`?

**Comment by grubles** [comment]
I went ahead and just renamed the rest. We can open another PR to remove that stuff in the future.

**Comment by YusukeShimizu** [comment]
Yes, I agree.
 I will create it separately.

**Pull request merged**